status-im/status-legacy
# Status go/integrate permissions api

### Summary

This moves the browser history and dapp permissions data from realm db in status-react to sqlite in status-go

### Testing notes
- only the browser and dapp permissions data are affected
- old browser and dapp permissions data is deleted

#### Platforms
Same effects on all platforms

### Steps to test

- Open Status
- login, visit different websites/dapps, also some that require permissions such as simpledapp.eth
- logout
- login again, history should still be there and accepted permissions shouldn't be asked again

status: ready
